misc/state-scores.json

```diff
@@ -0,0 +1 @@
+[{"state": "Alabama", "score": 1}, {"state": "Alaska", "score": 2}, {"state": "Arkansas", "score": 5}, {"state": "Arizona", "score": 0}, {"state": "California", "score": 4}, {"state": "Colorado", "score": 0}, {"state": "Connecticut", "score": 4}, {"state": "Delaware", "score": 4}, {"state": "Florida", "score": 1}, {"state": "Georgia", "score": 2}, {"state": "Hawaii", "score": 0}, {"state": "Iowa", "score": 0}, {"state": "Idaho", "score": 5}, {"state": "Illinois", "score": 1}, {"state": "Indiana", "score": 2}, {"state": "Kansas", "score": 4}, {"state": "Kentucky", "score": 1}, {"state": "Louisiana", "score": 1}, {"state": "Maine", "score": 4}, {"state": "Maryland", "score": 5}, {"state": "Massachusetts", "score": 2}, {"state": "Michigan", "score": 0}, {"state": "Minnesota", "score": 5}, {"state": "Missouri", "score": 2}, {"state": "Mississippi", "score": 5}, {"state": "Montana", "score": 1}, {"state": "North Carolina", "score": 3}, {"state": "North Dakota", "score": 1}, {"state": "Nebraska", "score": 1}, {"state": "New Hampshire", "score": 4}, {"state": "New Jersey", "score": 0}, {"state": "New Mexico", "score": 3}, {"state": "Nevada", "score": 1}, {"state": "New York", "score": 3}, {"state": "Ohio", "score": 3}, {"state": "Oklahoma", "score": 3}, {"state": "Oregon", "score": 4}, {"state": "Pennsylvania", "score": 3}, {"state": "Rhode Island", "score": 3}, {"state": "South Carolina", "score": 4}, {"state": "South Dakota", "score": 0}, {"state": "Tennessee", "score": 3}, {"state": "Texas", "score": 3}, {"state": "Utah", "score": 5}, {"state": "Virginia", "score": 5}, {"state": "Vermont", "score": 0}, {"state": "Washington", "score": 4}, {"state": "Wisconsin", "score": 2}, {"state": "West Virginia", "score": 1}, {"state": "Wyoming", "score": 2}]
```

Comment:
NIT: Can we run prettier, beautify, or something on this file to make it a little easier on the eyes?

Reply:
I can actually just remove this is you prefer - it's just the output of the script I wrote to generate fake data to dev on which I copied into public/data/ (and got prettified automatically)